contrib/zpages/src/main/java/io/opencensus/contrib/zpages/StatszZPageHandler.java

```diff
@@ -294,7 +294,7 @@ private static void emitDirectoryHeader(String path, PrintWriter out, Formatter
   private static void emitViewData(
       /*@Nullable*/ ViewData viewData, View.Name viewName, PrintWriter out, Formatter formatter) {
     if (viewData == null) {
-      formatter.format("<p class=\"view\">No Stats found for View: %s.</p>", viewName.asString());
```

Comment:
You don't need to change the class name. The test failed because of the additional colon.

Reply:
Okay, I'll leave the StatsZPageHandler as it is and hope the fix does it  :-) But if not, just let me know what I should do to fix.

Reply:
Oh, do you still need me to comment out that section in the Tester?

Reply:
No it's not needed as long as this line is fixed :)